apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/test/java/gov/cms/bfd/pipeline/ccw/rif/load/RifLoaderIT.java

```diff
@@ -698,6 +698,7 @@ public void loadSyntheticData() {
    * <p>This test ensures that changes to RifLoader that break compatibility with Synthea-generated
    * data are flagged.
    */
+  @Disabled
```

Comment:
Please add a comment with a link to the story we mentioned about fixing these up and re-enabling them once Synthea is updated for the new fields

Reply:
Updated